### PR TITLE
New package: gp2c-0.0.12

### PR DESCRIPTION
**Pull request by tornaria**
#### General
- [X] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [X] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

This is a compiler for GP scripts which typically run 3 or 4 times faster.

### CODE REVIEW

**Comment by tornaria** [comment]
Cross build failed, so I (hopefully) fixed that.

I don't know how to run checks in cross build mode. Is there some trick using qemu?.

**Comment by tornaria** [comment]
It seems a better fix is to use build-style=gnu-configure instead of configure. Then there is no need to check `$CROSS_BUILD` and mess with `$configure_args` in the template.

I noticed compilation for i686 is taking very long, it seems to be because pari-2.13.1 is not available for i686?

File: srcpkgs/gp2c/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'gp2c'
+pkgname=gp2c
+version=0.0.12
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```
Comment (FollieHiyuki) [inline srcpkgs/gp2c/template:6]: This is already the default.

**Comment by ericonr** [comment]
> I don't know how to run checks in cross build mode. Is there some trick using qemu?.

We don't run any such tests at the moment.

File: srcpkgs/gp2c/template (hunk 1 above)
Comment (tornaria) [reply srcpkgs/gp2c/template]: removed

**Pull request merged**